redbot/cogs/trivia/data/lists/music.yaml

```diff
@@ -0,0 +1,544 @@
+- Traffic
+Olly Peacock is the drummer with which group?:
+- Gomez
+In which American state is the museum called the Rock and Roll Hall of Fame?:
+- Ohio
+Which city was singer Craig David born in?:
+- Southampton
+Which US three piece band is made up of Taylor, Isaac and Zac?:
+- Hanson
+Which famous singer was born David Jones in 1947?:
+- David Bowie
+Thom Yorke is the lead singer with which band?:
+- Radiohead
+What was the name of Sonia's number one hit single of 1989?:
+- You'll Never Stop Me Loving You
+The soundtrack to which film was the best selling album in the UK in 1992?:
+- The Bodyguard
+Mark Snow had a hit single in 1996 with the theme from which TV series?:
+- The X-Files
+- The X Files
+Which American rock band had a UK top ten hit single in 1987 with "Alone"?:
+- Heart
+Which of the following operas was not composed by Puccini? Madame Butterfly, Tosca or The Barber Of Seville?:
+- The Barber Of Seville
+Who had a top ten hit in 1987 called "Shattered Dreams"?:
+- Johnny Hates Jazz
+Which group had a 2001 hit with a cover version of Billy JoelÃƒÂ¢Ã¢â€šÂ¬Ã¢â€žÂ¢s `Uptown Girl`?:
+- Westlife
+With which group would you most associate singer Bobby Gillespie?:
+- Primal Scream
+Which R.E.M. song includes the line "Let's play twister, let's play risk"?:
+- Man On The Moon
+The Pet Shop Boys had a UK Christmas Number 1 in 1987 with a song that was co-written by country star, Willie Nelson. What was the name of the song?:
+- Always On My Mind
+What was Whitney Houston's first UK number one hit single?:
+- Saving All My Love For You
+What was Bob Marley's backing band called?:
+- The Whalers
+Which band`s first top ten single was "Love Is Life" in 1970?:
+- Hot Chocolate
+Who`s 1978 debut single was called "Sultans Of Swing"?:
+- Dire Straits
+Who played bass guitar in the Beatles?:
+- Paul McCartney
+Who had a top ten hit in May 1990 with "I Still Haven't Found What I'm Looking For"?:
+- The Chimes
+In which decade did the Eurovision Song contest begin?:
+- 1950s (1956)
+- 1950s
+- 1956
+Which of Michael Jackson`s albums sold most copies in the UK? "Thriller" or "Bad"?:
+- Bad
+In which year did Bucks Fizz win the European Song Contest?:
+- 1981
+Which famous singer-songwriter won an Academy Award in 2001 for the song "Things Have Changed" from the film "Wonder Boys"?:
+- Bob Dylan
+Which rock star did Joan Rivers describe as being "the only man with child-bearing lips"?:
+- Mick Jagger
+What was the title of Michael Ball's song for Britain in the 1992 Eurovision Song Contest?:
+- One Step Out Of Time
+Who has had albums called "Blonde On Blonde", "Blood On The Tracks" and "Highway 61 Revisited"?:
+- Bob Dylan
+Which group sang the song "Take My Breath Away" which featured in the film "Top Gun"?:
+- Berlin
+What was the title of the Sex Pistol's first British hit single?:
+- Anarchy in the UK
+Which female vocalist released the album "Drag"?:
+- K. D. Lang
+What brand of beer does Homer Simpson drink?:
```

Comment:
How is line 430 and 431 related to music?

Reply:
It's related to a song named "[The Homer Simpson Beer Song](https://www.youtube.com/watch?v=TyiTY_jubF8)", but then it's upto you if you'd want me to keep it there ot just wipe it off